app/components/participation-list/mandataris-row.hbs

```diff
@@ -4,7 +4,7 @@
   <td>{{moment-format @mandataris.isBestuurlijkeAliasVan.geboorte.datum 'DD/MM/YYYY'}}</td>
   <td>
     <label class="checkbox checkbox--block">
-      <Input class="checkbox__toggle" @type="checkbox" @checked={{this.selected}} @click={{this.toggle}} />
+      <input class="checkbox__toggle" type="checkbox" checked={{@selected}} {{on "input" (fn @toggle @mandataris @selected)}} />
```

Comment:
I don't really get why you pass selected here, is that so they have the previous value?

Reply:
No the selected state is fully managed in the  parent components, so this is the latest state
Note it went from Input (2-way binding) to input (1-way)

Reply:
I just forgot to remove the now defunct backing class

Reply:
but `@selected` wont be updated here yet right, because it's 1 way binding.  the actual value after change/input would be in event.target.selected (event will be curried into your function I believe). given it's a toggle function I was guessing the value wasn't used (and thus shouldn't be passed?), but if it is used I think it might do the inverse of what you want :).

Sidenote: I prefer the change event for checboxes but I think both input and change would work here.

Reply:
having had a look at the toggleParticipant method, it will work because you expect the previous and not the actual value to be passed. ok that makes sense, though not entirely what I'd expect.

Reply:
I still dont quite understand what you mean
Though originally I didnt pass the selected value up to the handler (cause I could just get it there from the map)
but as I was debugging another issue, this was one of the things that changed
didnt look out of place so I left it I suppose

Reply:
for a callback from an input field (which is what this is) I typically expect to receive the actual value of that input. in this case you pass the value before the input. 

- you pass `@selected=false` down to the mandataris-row. the checkbox has checked=false
- user clicks the checkbox, the checkbox is now checked
- the component calls the callback with (mandataris, false)
- you negate this false in your toggle function

So what I'd expect (but this is nitpicking) to see is either not passing the value when toggling (the parent has the old value already after all) or passing the new value (true in my example)